documentation/docs/verifiable_credentials/overview.md

```diff
@@ -1,46 +1,49 @@
 ---
 title: Verifiable Credentials Overview
 sidebar_label: Overview
-description: Verifiable Credentials are statements about the holder. They can be verified online or in person, and the holder decides who to share them with.
+description: Verifiable Credentials are statements about the holder. They can be verified online or in person and the holder decides who to share them with.
 image: /img/Identity_icon.png
 keywords:
 - verifiable
 - credentials
 - person
+- reference
 ---
 
-Credentials are statements about an entity, such as properties that the entity posseses or capabilities that they have (like drivers licences, passports, or a person's age). Verifiable Credentials (VCs) are statements (eg. Alice has a drivers licence) that can be cryptographically verified by a third party, either online or in person. Additionally, the holder of the VC decides what is shared and who it is shared with.
+Credentials are statements about an entity, such as properties that the entity possesses or capabilities that they have, like a driver's license, a passport, or a someone's age. Verifiable Credentials (VCs) are statements, like someone having a driver's license, that can be cryptographically verified by a third party, either online or in person. The holder of the VC can then decide what is shared and who it is shared with.
 
-There are several types of actors that play different roles in a verifiable credential system. We'll start with a common example of how things work in the world today using physical credentials and centralized databases, and outline the roles that various entities play in the Verifiable Credential system.
+There are several types of actors that play different roles in a verifiable credential system. Starting with a common example of how things work today that you will find below, this article explains how physical credentials and centralized databases are used while outlining the roles that various entities play in the Verifiable Credential system.
```

Comment:
I'd start the last sentence with `This article explains..` and end it with `starting with a common ...`